### ISSUE TITLE
Issue when utilising URLDecode functionality in 1.1.1 / 1.1.2

### ISSUE THREAD

**Issue by Guitaraholic** [NDietrich/CyberChef-for-Splunk#13]
Experiencing an issue with Deployment of the app in a SHC environment where we continue to get this error 

```
URL.js:3 
        
       Uncaught (in promise) ReferenceError: i18n_register is not defined
    at URL.js:3
    at 3dfa1eb9-0eea-4f62-9ddc-484e752c2d21:2
    at Array.forEach (<anonymous>)
    at self.loadRequiredModules (3dfa1eb9-0eea-4f62-9ddc-484e752c2d21:2)
    at 3dfa1eb9-0eea-4f62-9ddc-484e752c2d21:2
    at l (3dfa1eb9-0eea-4f62-9ddc-484e752c2d21:2)
    at Generator._invoke (3dfa1eb9-0eea-4f62-9ddc-484e752c2d21:2)
    at Generator.next (3dfa1eb9-0eea-4f62-9ddc-484e752c2d21:2)
    at r (3dfa1eb9-0eea-4f62-9ddc-484e752c2d21:2)
    at s (3dfa1eb9-0eea-4f62-9ddc-484e752c2d21:2)
```

and the following error are occuring when we add specific this specific operation 

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'URL Decode')
    at 0cb851f1-9132-4b4b-9ef0-99add8d29525:2
    at Array.map (<anonymous>)
    at e.<anonymous> (0cb851f1-9132-4b4b-9ef0-99add8d29525:2)
    at l (0cb851f1-9132-4b4b-9ef0-99add8d29525:2)
    at Generator._invoke (0cb851f1-9132-4b4b-9ef0-99add8d29525:2)
    at Generator.next (0cb851f1-9132-4b4b-9ef0-99add8d29525:2)
    at r (0cb851f1-9132-4b4b-9ef0-99add8d29525:2)
    at s (0cb851f1-9132-4b4b-9ef0-99add8d29525:2)
```

We can see the i18n.js file is successfully loaded from Spunk but can't seem to identify why it believes this function is undefined

**Comment by NDietrich** [comment]
Can you let me know the spl you used that generated this error , and if
possible, one data value you passed to the function.
Does this happen every time you run cyberchef, or only intermittently. Is
it only for the  url decode command, or others as well?
Thanks

**Comment by Guitaraholic** [comment]
Hay - thanks for the quick reply ! That JS error is when you attempt to load in the CyberChef UI itself ... when we run ANY command ( even the makeresults example ) we were originally getting the token error fixed in 1.1.2 but are now getting a parse_csv error 

`Error in 'cyberchef' command: Fatal Error in function modify_payload: csv-parse error on payload: parse_csv is not a function
`

i was going to raise a separate ticket for this error but they might possible be linked?

**Comment by NDietrich** [comment]
I think that's all part of the same issue. I'll work on a fix this weekend
hopefully,

**Comment by NDietrich** [comment]
this looks to be a duplicate of issue #12, so i'll close this issue and we can use that one to track it.
I've released an update that I hope fixes this issue: [v1.1.3](https://github.com/NDietrich/CyberChef-for-Splunk/releases/tag/v1.1.3).  It works fine on my system (i.e. I'm unable to generate the error, but someone with a more complex setup may have problems). Please test and let me know via issue #12
